pymc_experimental/model_builder.py

```diff
@@ -193,6 +256,34 @@ def sample_model(self, **kwargs):
         return idata
 
     def set_idata_attrs(self, idata=None):
+        """
+        Set attributes on an InferenceData object.
+
+        Parameters
+        ----------
+        idata : arviz.InferenceData, optional
+            The InferenceData object to set attributes on.
+
+        Raises
+        ------
+        RuntimeError
+            If no InferenceData object is provided.
+
+        Returns
+        -------
+        None
+
+        Examples
+        --------
+        >>> import arviz as az
```

Comment:
These are pretty weird examples, more like tests. This should give the user info on how to use the method to achieve something.

Reply:
Doctests provide an option to perform additional checks based on the docstrings, they might look a bit odd, but you're right, this one is not the best. I'll try to refactor it. Maybe putting some comment on the last line that is actually both doctest and the user manual (in this case 280) would help?